Dockerfile

```diff
@@ -20,7 +20,12 @@ COPY --from=build /usr/src/app/public/build/bundle.js /usr/share/nginx/html/buil
 RUN chmod a+r -R /usr/share/nginx/html
 
 COPY deploy/.docker/config.js.template /etc/nginx/config.js.template
+COPY deploy/.docker/error.html /usr/share/nginx/html/error.html
+COPY deploy/.docker/nginx.conf /etc/nginx/nginx.conf
 
+RUN sed -i 's|<base href="/">|<base href="/fta-fmea/">|g' /usr/share/nginx/html/index.html
```

Comment:
Why do we need to do that? Can't we just remove `<base ...` attribute? (not sure here as it depends I guess on react router related stuff, but I think that @LaChope did not need to do that. Am I right @LaChope?

Reply:
Indeed, I did not need that. @Matthew-Kulich you need to go in your `package.json` and add `"homepage": ".",`

Reply:
Ok, thanks

Reply:
Implemented.